images/build/Dockerfiles/gitHubRunners.BuildPackDepsStretch.Dockerfile

```diff
@@ -1,4 +1,2 @@
-# The digest here must match with what is present in the following location
-# so that downloading of the image is very fast in GitHub workflows
-# https://github.com/actions/virtual-environments/blob/master/images/linux/Ubuntu1804-README.md
-FROM buildpack-deps:stretch@sha256:34a18637ed801407f7a17a29575e82264fb0818f9b6a0c890f8a6530afea43dc
+# Auto-generated by Oryx-AutoUpdate build pipeline
+FROM buildpack-deps:stretch@sha256:82a686ba95fdf2bf4a5f5264e8e55b8aa272ffcedaed7826777f08de0d9e1146
```

Comment:
nit: Could you keep the comments previously in this file, feels we can keep it, easier to find the link

Reply:
Sure, sounds good